lib/public/views/LhcFills/ActiveColumns/lhcFillsActiveColumns.js

```diff
@@ -36,6 +39,12 @@ export const lhcFillsActiveColumns = {
             { fillNumber },
             [
                 h(CopyToClipboardComponent, { value: fillNumber, id: fillNumber }, 'Copy Fill Number'),
+                frontLink(
```

Comment:
The "Add log to this fill" button should be made to come after `infoLoggerLinksComponents` given this is now the layout in the environments overview page and add the plus icon too so we keep an even more consistent design. (Not to have known before completing ticket).